pandas/tests/io/test_sql.py

```diff
@@ -1613,6 +1613,7 @@ class _TestSQLAlchemy(SQLAlchemyMixIn, PandasSQLTest):
     flavor: str
 
     @pytest.fixture(autouse=True, scope="class")
+    @classmethod
```

Comment:
I think you need to swap these two for pytest collection to work
```suggestion
    @classmethod
    @pytest.fixture(autouse=True, scope="class")
```